### PR TITLE
spellchecks "cyro" -> "cryo"

### PR DESCRIPTION
**Pull request by MrDoomBringer**
:cl: MrDoomBringer
spellcheck: Changed "cyro" to "cryo" in a few item descriptions and flavortexts. 
/:cl:

also includes other minor formatting changes

### CODE REVIEW

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 1]
```diff
@@ -595,7 +595,7 @@
 	death = FALSE
 	faction = ROLE_SYNDICATE
 	outfit = /datum/outfit/snowsyndie
-	flavour_text = {"You are a syndicate operative recently awoken from cyrostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
+	flavour_text = {"You are a syndicate operative recently awoken from cryostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
 	disposed of swirfly to assure no gathered information is stolen or lost. Try not to wander too far from the outpost as the caves can be a deadly place even for a trained operative such as yourself."}
```
Comment (Irafas) [inline code/modules/awaymissions/mission_code/snowdin.dm]: While correcting spelling you might as well fix this one to how its supposed to be, 

> All intruders should be disposed of by swirly to assure no gathered information is stolen.

Comment (MrDoomBringer) [reply code/modules/awaymissions/mission_code/snowdin.dm]: lmfao

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 2]
```diff
@@ -595,8 +596,8 @@
 	death = FALSE
 	faction = ROLE_SYNDICATE
 	outfit = /datum/outfit/snowsyndie
-	flavour_text = {"You are a syndicate operative recently awoken from cyrostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
-	disposed of swirfly to assure no gathered information is stolen or lost. Try not to wander too far from the outpost as the caves can be a deadly place even for a trained operative such as yourself."}
+	flavour_text = "<span class='big bold'>You are a syndicate operative recently awoken from cryostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
```
[the remark this answers is not in the document]
Comment (SpaceManiac) [reply code/modules/awaymissions/mission_code/snowdin.dm]: also you need to put the `{" "}` back or combine these onto one line

[the remark this answers is not in the document]
Comment (MrDoomBringer) [reply code/modules/awaymissions/mission_code/snowdin.dm]: couldnt i just do \

[the remark this answers is not in the document]
Comment (MrDoomBringer) [reply code/modules/awaymissions/mission_code/snowdin.dm]: like we do everywhere else in the code

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**